test/core/2_pendleData.spec.ts

```diff
@@ -1,23 +1,34 @@
 import { expect } from 'chai';
-import { Contract } from 'ethers';
-import { evm_revert, evm_snapshot } from '../helpers';
+import { Contract, FixedNumber,  BigNumber as BN } from 'ethers';
+
+import { consts, evm_revert, evm_snapshot, tokens } from '../helpers';
 import { marketFixture, MarketFixture } from './fixtures';
+import PendleData from '../../build/artifacts/contracts/core/PendleData.sol/PendleData.json';
+
 const { waffle } = require('hardhat');
 
-const { loadFixture } = waffle;
+const { provider, deployContract, loadFixture } = waffle;
 
 describe('PendleData', async () => {
   let data: Contract;
   let treasury: Contract;
+  let fixture: MarketFixture
+  let pausingManager: Contract;
+  let testToken: Contract;
   let snapshotId: string;
   let globalSnapshotId: string;
 
+  const wallets = provider.getWallets();
+  const [alice, bob, charlie] = wallets;
+
   before(async () => {
-    const fixture: MarketFixture = await loadFixture(marketFixture);
+    fixture = await loadFixture(marketFixture);
```

Comment:
There are functions for parsing the fixtures to test environment (TestEnv) - storing all your contracts below.